zorg/jenkins/jobs/jobs/clang-3.4-debuginfo-statistics

```diff
@@ -76,7 +89,7 @@ pipeline {
                 withCredentials([string(credentialsId: 's3_resource_bucket', variable: 'S3_BUCKET')]) {
                     sh """
                         source ./venv/bin/activate
-                        echo "ARTIFACT=${params.ARTIFACT}"
+                        export $(grep ARTIFACT last_good_build.properties)
```

Comment:
I this still downloading the toolchain package?

Reply:
Yes, it should. It points the `ARTIFACT` to the one listed in `last_good_build.properties` and that is fetched in the next line.